docs/current/cookbook.md

```diff
@@ -994,6 +994,44 @@ The following code listing demonstrates how to add multiple environment variable
 </TabItem>
 </Tabs>
 
+### Organize pipeline code into modules
+
+The following code listing demonstrates how to organize Dagger pipeline code into independent modules (or functions/packages, depending on your programming language) to improve code reusability and organization. It also demonstrates how to reuse the Dagger client and, therefore, share the Dagger session between modules.
+
+:::note
+Using multiple Dagger clients in the same pipeline can result in unexpected behavior. It is recommended to reuse the Dagger client wherever possible, instead of creating a new client.
```

Comment:
```suggestion
Initializing and using multiple Dagger clients in the same pipeline can result in unexpected behavior. It is recommended to reuse the Dagger client wherever possible, instead of creating a new client.
```